src/pages/search/advanced_search/ScientificValueFilter.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Checkbox, FormControlLabel } from '@mui/material';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ResultParam } from '../../../api/searchApi';
+
+enum ScientificValueLevels {
+  LevelZero = 'Unassigned,LevelZero',
+  LevelOne = 'LevelOne',
+  LevelTwo = 'LevelTwo',
+}
+
+export const ScientificValueFilter = () => {
+  const [t] = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const scientificValueParam = searchParams.get(ResultParam.ScientificValue) || '';
+
+  const [selectedScientificValues, setSelectedScientificValues] = useState({
+    levelZero: scientificValueParam.includes(ScientificValueLevels.LevelZero),
+    levelOne: scientificValueParam.includes(ScientificValueLevels.LevelOne),
+    levelTwo: scientificValueParam.includes(ScientificValueLevels.LevelTwo),
+  });
+
+  // Used to clear value based on URL params
+  useEffect(() => {
+    setSelectedScientificValues({
+      levelZero: scientificValueParam.includes(ScientificValueLevels.LevelZero),
+      levelOne: scientificValueParam.includes(ScientificValueLevels.LevelOne),
+      levelTwo: scientificValueParam.includes(ScientificValueLevels.LevelTwo),
+    });
+  }, [scientificValueParam]);
```

Comment:
hva trengs denne bolken til?

Reply:
Var for at "Nullstill Valg" skulle fungere, men trengs ikke nå som det ikke brukes `useState`